setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='1.2.0',
+    version='2.0.0',
```

Comment:
Sure.  Half of our repos are languishing in pre-1.0 land despite being in heavy use in production.  I guess I just don't get the aversion to bumping the version number appropriately.  Given the way we work, we should have a lot more v13.2.1 level version numbers, and I think that's perfectly fine.